kuma/api/v1/tests/test_views.py

```diff
@@ -37,7 +37,7 @@ def test_whoami_disallowed_methods(client, http_method):
 def test_whoami_anonymous(client, settings):
     """Test response for anonymous users."""
     # Create some fake waffle objects
-    Flag.objects.create(name="section_edit", authenticated=True)
+    Flag.objects.create(name="vip_only", authenticated=True)
```

Comment:
Nice attention to detail in maintaining the check for the `authenticated=True` case!